bigbluebutton-html5/client/compatibility/kurento-extension.js

```diff
@@ -480,6 +480,9 @@ window.getScreenConstraints = function (sendSource, callback) {
     // now invoking native getUserMedia API
     callback(null, screenConstraints);
   } else if (isSafari) {
+    document.dispatchEvent(new Event('safariScreenshareNotSupported'));
+    return;
```

Comment:
There is code remaining after the `return;` Could you please either remove it if obsolete, comment out if the toast is a temporary solution, or rework the if-else?